src/read_config_file.c

```diff
@@ -699,8 +699,8 @@ static PetscErrorCode ParseFlowConditions(yaml_event_t *event, YamlParserState *
       } else {
         if (!strcmp(state->parameter, "type")) {
           PetscInt selection;
-          SelectItem(value, 3, (const char *[3]){"dirichlet", "neumann", "reflecting"},
-                     (PetscInt[3]){CONDITION_DIRICHLET, CONDITION_NEUMANN, CONDITION_REFLECTING}, &selection);
+          SelectItem(value, 4, (const char *[4]){"dirichlet", "neumann", "reflecting", "critical-outflow"},
```

Comment:
@jeff-cohere  I used `critical-outflow` instead of using `critical_outflow`, which I think is the correct approach to avoid confusing yaml, but I want to double-check with you if it is the correct thing to do.